src/core/public/chrome/nav_links/nav_links_service.ts

```diff
@@ -99,20 +99,20 @@ export class NavLinksService {
   private readonly stop$ = new ReplaySubject(1);
 
   public start({ application, http }: StartDeps): ChromeNavLinks {
+    const legacyAppLinks = application.availableLegacyApps.map(
```

Comment:
will this service work with New Platform apps?

Reply:
Yes, but NP apps won't exist until #41007. At that point the API will be available for NP apps.